pkgs/top-level/python-packages.nix

```diff
@@ -7341,10 +7341,10 @@ in {
 
   vsts-cd-manager = callPackage ../development/python-modules/vsts-cd-manager { };
 
-  vtk = toPythonModule (pkgs.vtk_7.override {
+  vtk_9 = disabledIf isPy27 (toPythonModule (pkgs.vtk_9.override {
```

Comment:
Please move this into package. ``disabled = isPy2k``.

Reply:
> Please move this into package. disabled = isPy2k.

This is only done with `buildPython*` packages, not other builders. Technically it could be done, but it would force a rebuild for the top-level package.